ert_shared/port_handler.py

```diff
@@ -134,7 +138,17 @@ def get_family(host: str) -> socket.AddressFamily:
         return socket.AF_INET
 
 
+# See https://stackoverflow.com/a/28950776
 def _get_ip_address() -> str:
-    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-    s.connect(("8.8.8.8", 80))
-    return s.getsockname()[0]
+    try:
+        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        s.settimeout(0)
+        s.connect(("10.255.255.255", 1))
```

Comment:
What is this address? We're all familiar with 8.8.8.8, but maybe this needs a comment?

Reply:
will do